test/state/state.hpp

```diff
@@ -125,7 +136,7 @@ struct Log
 
 struct TransactionReceipt
 {
-    Transaction::Kind kind = Transaction::Kind::legacy;
+    Transaction::Type kind = Transaction::Type::legacy;
```

Comment:
should this be `type`?
```suggestion
    Transaction::Type type = Transaction::Type::legacy;
```

Reply:
Likely.

Reply:
Done.